src/bot/config.py

```diff
@@ -0,0 +1,42 @@
+import os
+
+from dotenv import load_dotenv
+
+load_dotenv()
+
+
+# Token
+BOT_TOKEN = os.getenv("BOT_TOKEN")
+assert BOT_TOKEN is not None
+
+# Developer options
+TESTING_MODE = bool(os.getenv("TESTING_MODE"))
+TESTING_BOT_TOKEN = os.getenv("TESTING_BOT_TOKEN") or BOT_TOKEN
+if TESTING_MODE:
+    assert TESTING_BOT_TOKEN is not None
+
+# IDs
+OWNER_IDS: tuple = (
+	534651911903772674, # GetPsyched
+	330028725553070090, # meatball
+	1063920139713654825, # malik
+	746807967650807810, # Jezz
+	707977917485023376, # HiPeople
+)
+
+# PostgreSQL
+class DB:
```

Comment:
I dont how relevant this is thinking @malik1004x talked about only having a single model object.